case-lib/opt.sh

```diff
@@ -176,7 +176,7 @@ func_opt_parse_option()
             [[ "$TPLG" ]] && printf "TPLG=%s" "$TPLG "
             printf "%s %s\n" "$SCRIPT_NAME" "$SCRIPT_PRAM"
             printf 'Commit:\n'
-            git -C "$SCRIPT_HOME" log --oneline --decorate -n 5
+            git -C "$SCRIPT_HOME" log --oneline --decorate -n 5 | true
```

Comment:
```suggestion
            git -C "$SCRIPT_HOME" log --oneline --decorate -n 5 || true
```
I'm sorry my earlier typo mislead you.

Compare:
```shell
printf foo | true
printf foo || true
```